.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: C
+
+before_install:
+    - (cd ..; hg clone https://hg.mozilla.org/projects/nspr)
+
+script:
+    - make nss_build_all
+    - cd tests; NSS_TESTS=ssl_gtests NSS_CYCLES=standard ./all.sh 
```

Comment:
This is going to run more than I think you really want to run.  Can you just run the ssl_gtests ?